test/buttercup-tests/flycheck/flycheck-test.el

```diff
@@ -0,0 +1,23 @@
+(describe "Flycheck integration"
+  (it "integrates flycheck with omnisharp"
+    (ot--open-the-minimal-solution-source-file "FlycheckTest.cs")
+    (ot--buffer-contents-and-point-at-$
+     "$public class MyClass {
+         DoesNotExist foo;
+       }")
+
+    (flycheck-mode)
+    (flycheck-list-errors)
+    ;; (ot--wait-for (lambda() (print "waiting")(get-buffer "*Flycheck errors*")) 5)
+    (ot--wait-for (lambda() (and (boundp 'flycheck-current-errors)
+                                 (> (length flycheck-current-errors) 0))) 10)
```

Comment:
This line never finds any errors when ran from the integration tests script - I can't figure out why. I tried quite a few other ways of achieving the same thing but no luck. 
It's odd - the commented out look for 'Flycheck buffer' line above doesn't fail, so I can see that flycheck mode is running.

The test passes every time when ran locally using `M-x buttercup-run-at-point`

@sp3ctum Any chance that you could take a look at this for me? I'd appreciate it.